README.rst

```diff
@@ -41,25 +41,33 @@ you access to the API client that you want:
 
     import signalfx
 
-    sfx = signalfx.SignalFx()
+    sfx = signalfx.SignalFx(api_endpoint='https://api.{REALM}.signalfx.com',
+            ingest_endpoint='https://ingest.{REALM}.signalfx.com',
+            stream_endpoint='https://stream.{REALM}.signalfx.com')
+
 
     # For the ingest API
-    ingest = sfx.ingest('ORG_ACCESS_TOKEN')
+    ingest = sfx.ingest('ORG_TOKEN')
 
     # For the REST API
-    rest = sfx.rest('USER_ACCESS_TOKEN')
+    rest = sfx.rest('API_TOKEN')
```

Comment:
API_TOKEN  means it is required (you can't use an org token). ACCESS_TOKEN means you can use either org or user API token. 

If this is for authenticating into the API, I think this is correct. Otherwise, it should probably be ACCESS_TOKEN. I can't tell from the context. Ignore if this is correct as is.

https://developers.signalfx.com/administration/access_tokens_overview.html